book/intro.md

```diff
@@ -1,41 +1,41 @@
+## How to get the most out of the book
 
-## About us
+This guidance describes software engineering good practices that are tailored to those working with data using code. It is designed for those who would like to quality assure their code and increase the reproducibility of their analyses. Software that apply these practices are referred to as reproducible analytical pipelines (RAP).
 
-The Methods and Quality directorate supports government analysis by providing guidance, consultancy and training.
+ This guidance is relevant if you are:
+- writing code to automate part of your work and would like to assure that it is working as expected
+- developing statistical or data engineering pipelines and would like to assure that they are sustainable and reproducible
+- developing models and would like to assure that they are transparent and reproducible
+- developing data science techniques and would like your code to be useful to others
+- looking for a high level introduction to software engineering practices in the context of analysis and research
 
-You can find us on:
-- Our websites: [Government Analysis Function](https://www.gov.uk/government/organisations/government-analysis-function) and [Government Statistical Service](https://gss.civilservice.gov.uk/)
-- Email: [analysis.function@ons.gov.uk](mailto:analysis.function@ons.gov.uk) and [gsshelp@statistics.gov.uk](mailto:gsshelp@statistics.gov.uk)
-- Slack: [Government Data Science](https://govdatascience.slack.com) and [GSS](https://gov-stats-service.slack.com)
-- Twitter: [Government Analysis Function](https://twitter.com/gov_analysis),  [GSS Good Practice Team](https://twitter.com/gssgoodpractice) and [UK GSS](https://twitter.com/ukgss)
+The good practices outlined in the book are general to many applications of programming, so may also be relevant for those outside of government.
 
-## Citing the book
+The [RAP learning pathway](https://learninghub.ons.gov.uk/totara/program/view.php?id=6) provides training for many of these good practices. This book can be used to guide your learning and as a reference when applying these practices in your work. Each chapter describes the risks that each practice may help to address. As recommended by [the Aqua book](https://www.gov.uk/government/publications/the-aqua-book-guidance-on-producing-quality-analysis-for-government), you should strive to apply the most appropriate quality assurance practices given the risks associated with your work.
 
-The following structure can be used to reference the current version of the book:
+The principles in this book are language agnostic. The book does not aim to form a comprehensive learning resource and you may often need to study further resources to implement these practices. That said, examples and useful references are provided for **Python** and **R**, as open source languages that are commonly applied across government.
```

Comment:
Do we want to say Python and R are the recommended languages, following Martin's presentation/paper?